popupModal.css

```diff
@@ -0,0 +1,433 @@
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #f7f7f9;
+  overflow-x: hidden;
+}
+
+button {
+  cursor: pointer;
+}
+
+.parent_main {
+  filter: none;
+}
+
+ul {
+  list-style: none;
+}
+
+.blur {
+  filter: blur(20px);
+}
+
+.vanish {
+  display: none;
+}
+
+.card-contents {
+  display: grid;
+  row-gap: 88px;
+  justify-content: center;
+  padding: 114px 24px;
+  list-style-type: none;
+}
+
+.modal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: #c1c7d0;
+}
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided.
- There are some top and bottom margins in the Figma template that are not in your design. Kindly add these margins. :pray: :pray: 
Your Design
![image](https://user-images.githubusercontent.com/98363075/202856180-51a2486f-3a5e-4c63-9ed6-3d12374a1b0b.png)

Figma Design
![image](https://user-images.githubusercontent.com/98363075/202856316-0df708eb-bf96-4614-be42-0f6ff6569c24.png)

- Kindly add the black bottom line in the mobile version. :eyes: :eyes: 
Your design
![image](https://user-images.githubusercontent.com/98363075/202856465-7023a56a-4168-4dd4-a536-053ea8c2e3ee.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/202856423-d03d6fcf-405b-4fc1-bc50-677d93883c0a.png)